test/expected-output-test/arrays/input.ddlog

```diff
@@ -0,0 +1,7 @@
+R(x text[]).
+
+S(x text,
+  y text[]).
+
+T(x[3], z) :- R(x), S(z, y), x[1] = y[1], x[2] = y[2].
```

Comment:
Then can you add examples of using compound expressions, not just names and integer literals?